docs/source/guide/index.rst

```diff
@@ -47,13 +47,14 @@ It executes administrative commands to the Repository Service for TUF.
 
     Administrative Commands
 
-    ╭─ Options ────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
-    │ --help  -h    Show this message and exit.                                                                                                                                                                                    │
-    ╰──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
-    ╭─ Commands ───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
-    │ ceremony                                              Start a new Metadata Ceremony.                                                                                                                                         │
-    │ import-targets                                        Import targets to RSTUF from exported CSV file.                                                                                                                        │
-    ╰──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
+    ╭─ Options ────────────────────────────────────────────────────────────────────╮
+    │ --help  -h    Show this message and exit.                                    │
+    ╰──────────────────────────────────────────────────────────────────────────────╯
+    ╭─ Commands ───────────────────────────────────────────────────────────────────╮
+    │ ceremony                    Start a new Metadata Ceremony.                   │
+    │ import-artifacts            Import artifacts to RSTUF from exported CSV file.│
```

Comment:
```suggestion
    │ import-artifacts            Import artifacts to RSTUF from exported CSV file. │
```

Reply:
If you do that you need to add a space to the other rows as well.

Reply:
I think we can keep it. It is just an example to the user in the docs., unless there are other changes to do.